src/content/documentation/develop/request-the-right-permissions.md

```diff
@@ -150,7 +150,7 @@ This section looks at situations where you might be asking for more permissions
 
 ### Ask for only the permissions your extension uses
 
-This may seem obvious, but if you create an extension by using an earlier example as a template or you remove a feature during development or testing, you may be asking for permissions your extension doesn't need. Addressing this is a case of doing a manual check of your code against the permissions (`"permissions"` and ["optional_permissions"](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_permissions)) that you're requesting in the extension's manifest.json.
+This may seem obvious, but if you create an extension by using an earlier example as a template or you remove a feature during development or testing, you may be asking for permissions your extension doesn't need. Addressing this is a case of doing a manual check of your code against the permissions (`"permissions"` and ["optional_permissions"](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_permissions)) that you're requesting in the extension's `manifest.json`.
```

Comment:
Not introduced here, but it could be good if permissions could link to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions

Also `optional_permissions` should be in backticks.

I think the double quotes could be dropped for both `permissions` and `optional_permissions`.